src/datasources/transaction-api/transaction-api.service.spec.ts

```diff
@@ -173,4 +173,41 @@ describe('TransactionApi', () => {
       expect(httpErrorFactory.from).toHaveBeenCalledTimes(1);
     });
   });
+
+  describe('Modules', () => {
+    it('should return Safes with module enables', async () => {
```

Comment:
typo:

```suggestion
    it('should return Safes with module enabled', async () => {
```

Reply:
Fixed in 7655818de0c1138f1f72ba1f6b89921f926fe3b0.